less/notificationBar.less

```diff
@@ -7,7 +7,19 @@
 
 .notificationBar {
   display: inline-block;
+  position: relative;
+  z-index: 1000;
```

Comment:
I'd change this into `@zindexNavigationBar` according to https://github.com/brave/browser-laptop/blob/ce25254d5bb0a391ffa5ebbc0d5f70ef2348155a/less/variables.less#L129